src/utils/refresh.js

```diff
@@ -1,7 +1,11 @@
 import { watchLocalStorage } from './watchers'
 import { eventHub } from './eventHub'
 
-window.localStorage.setItem('reloadConsole', 'false')
+try {
+    window.localStorage.setItem('reloadConsole', 'false')
+} catch (e) {
+    
```

Comment:
can we log the error here in case it does fail?

Reply:
added: console.log(e)